indexer/cli/cli.go

```diff
@@ -63,16 +64,21 @@ func runApi(ctx *cli.Context) error {
 	logger := log.NewLogger(log.ReadCLIConfig(ctx))
 
 	configPath := ctx.String(ConfigFlag.Name)
-	cfg, err := config.LoadConfig(configPath)
+	conf, err := config.LoadConfig(configPath)
 	if err != nil {
 		logger.Error("failed to load config", "err", err)
 		return err
 	}
 
-	fmt.Println(cfg)
+	db, err := database.NewDB(conf.DB)
+
+	if err != nil {
+		logger.Crit("Failed to connect to database", "err", err)
+	}
+
+	server := api.NewApi(db.BridgeTransfers, logger)
 
-	// finish me
-	return err
+	return server.Listen(strconv.Itoa(conf.API.Port))
```

Comment:
LGTM! But similar to `runIndexer`, we probably want catch interrupts/panics and gracefully handle them

Reply:
feel free to ignore this for now and handle this later

Reply:
```suggestion
        // FIXME similar to runIndexer, we probably want catch interrupts/panics and gracefully handle them
	return server.Listen(strconv.Itoa(conf.API.Port))
```